Source/WebKit/WebProcess/WebPage/gtk/AcceleratedSurfaceDMABuf.cpp

```diff
@@ -166,7 +166,7 @@ std::unique_ptr<AcceleratedSurfaceDMABuf::RenderTarget> AcceleratedSurfaceDMABuf
     };
 
 #define ADD_PLANE_ATTRIBUTES(planeIndex) { \
-    fds.append(UnixFileDescriptor { gbm_bo_get_fd_for_plane(bo, planeIndex), UnixFileDescriptor::Adopt }); \
+    fds.append(UnixFileDescriptor { gbm_bo_get_fd_for_plane2(bo, planeIndex), UnixFileDescriptor::Adopt }); \
```

Comment:
This is 2 is a bug I introduced by mistake to test the option I don't have, it should be removed from GBMVersioning.h